lib/active_merchant/billing/gateways/orbital/orbital_codes.rb

```diff
@@ -0,0 +1,330 @@
+module ActiveMerchant
+  module Billing
+    module OrbitalCodes
+      CURRENCY_CODES = {
```

Comment:
Reviewing the PDF documentation, I find that the following currencies need to be included. Please verify and add them:
```suggestion
      CURRENCY_CODES = {
       #...existing code...
       'AMD' => '051',   # Armenian Dram
       'GWP' => '624',   # Guinea-Bissau Peso
       'MGF' => '450',   # Malagasy Franc
       #...existing code...
     }

     CURRENCY_EXPONENTS = {
       #...existing code...
       'AMD' => 2,   # Armenian Dram
       'GWP' => 2,   # Guinea-Bissau Peso
       'MGF' => 2,   # Malagasy Franc
       #...existing code...
     }
```

Reply:
done.